content/e/ppw/2024-10-26-ppw-co-za-noc.md

```diff
@@ -14,7 +14,22 @@ city = "Warszawa"
 
 "Co Za Noc" (_What a Night_) was announced on 10.09.2024, set to take place in a brand new location - a post-industrial complex at Mińska 65 street.
 
-{{ skip_card() }}
+{% card(predicted=true) %}
+- - 'Zmowa przeciwko zmowie'
+  - 'Zmowa'
```

Comment:
This adds both as characters in the people index. Instead, make this a team, with unspecified members:

```suggestion
- - 'Zmowa przeciwko zmowie: ???, ???, ???, ???, ???'
  - 'Zmowa: ???, ???, ???, ???, ???'
```